release/release.md

```diff
@@ -0,0 +1,122 @@
+# How to do a Kata Containers Release
+
+* [Introduction](#Introduction)
+* [Requirements](#Requirements)
+* [Release process](#release-process)
+
+# Introduction
+
+If you are reading this document, you might want to create a Kata Containers
+Release.
+
+The Kata Containers Release Process is defined in the following
+[document][release-process-definition]. To simplify this process, we have
+created a [Release Checklist][release-checklist]. Finally, to simplify the
+Release Checklist process we have automated most of the process, this document
+guides on how to use release scripts instead of do all the checklist manually.
+
+
+## Requirements
+
+- It is recommended to have at least 2 GB of free disk space to perform these tasks.
+
+- [Go](https://golang.org/doc/install#)
+
+- [hub](https://github.com/github/hub)
+  If you use GitHub 2FA, configure the hub manually to create a file ~/.config/hub
+  ```yaml
+  github.com:
+  - user: YOUR_USER
+  oauth_token: TOKEN
+  protocol: https
+  ```
+- OBS account with permissions on /home:katacontainers (https://build.opensuse.org/project/subprojects/home:katacontainers)
```

Comment:
@marcov - is there a way to describe these OBS permissions briefly do you know? I personally find the OBS website pretty difficult to use.

Reply:
> - An account on https://build.opensuse.org with maintainer rights for the [home:katacontainers project](https://build.opensuse.org/project/show/home:katacontainers). Maintainer rights can be asked with the "Request Role Addition" link in the project homepage.